spring-batch/src/test/java/org/baeldung/conditionalflow/model/NumberInfoUnitTest.java

```diff
@@ -6,9 +6,9 @@
 
 import org.junit.jupiter.api.Test;
 import org.junit.runner.RunWith;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.junit4.SpringRunner;
 
-@RunWith(SpringJUnit4ClassRunner.class)
+@RunWith(SpringRunner.class)
```

Comment:
Let's try to use JUnit5 instead of 4. Replace with @ExtendWith(SpringExtension.class)

Reply:
Done